kafe2/fit/_base/plot.py

```diff
@@ -631,7 +631,7 @@ def plot_residual(self, target_axes, error_contributions=("data",), **kwargs):
             **kwargs,
         )
 
-    def plot_pull(self, target_axes, error_contributions=("data",), **kwargs):
+    def plot_pull(self, target_axes, error_contributions=('model',), **kwargs):
```

Comment:
Why are you changing this default? In kafe2 errors can be assigned as "data" or "model" errors, I think for pull plots both should be used. Theoretically this would then mean specifying `("data", "model")` here. But looking at the code `"data"` seems to be mapped to `XYFit::y_total_error` so this would be double counting the model errors. @cverstege what is the intended interpretation of `data_yerr` and `model_yerr` in the plot adapters? Is it supposed to just be a 1:1 mapping to the corresponding fit properties or do they have a different meaning?

Reply:
I talked to Günter, the pull should be normalized to the data errors.